certified-connectors/Cornerstone Learning vILT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1143 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Cornerstone Learning vILT",
+		"version": "1.0",
+		"description": "The custom connector streamlines vILT integration by leveraging customers' Azure AD tenant environments, ensuring data integrity and simplifying authentication. No external Graph API permissions are needed, enhancing security. This versatile connector facilitates low-code solutions for diverse scenarios. By incorporating this custom connector, the full potential of Microsoft Teams API in the case of MS Teams integration is unlocked seamlessly within customers' infrastructure.",
+		"contact": {
```

Comment:
Please add the url to contact, refer this template - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json

Reply:
I understand that you are referring to the email tag? If so, I have added as value NA, we don't have a support email, users have to log a ticket in our platform.

Reply:
Kindly add email tag as well, this is needed for future communication to add into our internal tool.